drivers/led/sn32f2xx.c

```diff
@@ -102,25 +84,28 @@ static uint8_t last_key_col                            = 0;   // key col scan co
 static uint8_t led_duty_cycle[SN32_RGB_MATRIX_ROWS_HW] = {0}; // track the channel duty cycle
 #    endif
 #endif
-#if (DIODE_DIRECTION == ROW2COL)
+#if ((DIODE_DIRECTION == ROW2COL) && (SN32_PWM_DIRECTION != DIODE_DIRECTION))
```

Comment:
In my ROW2COL board this is not compiling because row_shifter is not definied further down.
When I change it to == it works.